### DISCUSSION TITLE
launchpad x velocity sensitivity In force mode

### DISCUSSION THREAD

**Discussion by varasone** [TheKikGen/MPCLiveXplore-libs#23] [General]
Hello, I am thinking of buying a launchpad x to use with i am force and I would like to know the speed sensitivity of the pads, does it work? Because if the pressure sensitivity doesn't work I would buy a launchpad mini mk3, please guide me

**Comment by TheKikGen** [comment]
![image](https://github.com/TheKikGen/MPCLiveXplore-libs/assets/35703467/7e11a1ab-efee-417c-bd61-d63d47b9b457)
In the Launchpad X documentation you can download online.

**Reply by varasone** [reply]
Thank you very much, but the question was if you send and receive those messages on the mpc, thank you

**Reply by TheKikGen** [reply]
The Velocity is sent with the NOTE ON midi message. The MPC (and IamForce) will interpret that correctly, as any midi keyboard.

**Reply by varasone** [reply]
Thanks
